pkgs/top-level/python-packages.nix

```diff
@@ -8493,7 +8493,18 @@ in {
 
   resampy = callPackage ../development/python-modules/resampy { };
 
-  resolvelib = callPackage ../development/python-modules/resolvelib { };
+  resolvelib-0_5 = callPackage ../development/python-modules/resolvelib {
```

Comment:
The way we normally name attributes this would be `resolvelib_0_5`

Reply:
But also having multiple versions of a single package violates the `python-packages.nix` policy.

Reply:
Right , but there's only Ansible and pdm (once the PR lands) that use it and they are both applications. 

Alternatively we'd have to mark Ansible broken or fix it upstream to land pdm.

I can move the Ansible version into the Ansible package if that would help.

Reply:
Ansible is a library, since it is consumed by 

- ansible-runner
- ansible-lint
- pytest-ansible

Reply:
@fridh does it make sense to move the newer version of resolvelib into the pdm directory for now?

Reply:
You can use `packageOverrides` to override the resolvelib version in `python3Packages` if you're using `buildPythonApplication`.